src/index.ts

```diff
@@ -30,7 +31,7 @@ export type {
   MatchedRoute,
   RouteResource,
   RouteContext,
-  RouterStoreContext,
+  RouterContext,
```

Comment:
same comment here. can we export both for now?

Reply:
If we don't break the API now, how do we plan migration late on? This is 0.x and I'd like having several APIs cleaned up before v1. I think doing bit by bit with a bunch of small releases is better than a big 1.0 break.

Reply:
okay no problem. Happy with this change :)